docs/resources/synthetics_test.md

```diff
@@ -609,8 +609,9 @@ Optional:
 - `no_saving_response_body` (Boolean) Determines whether or not to save the response body.
 - `number_of_packets` (Number) Number of pings to use per test for ICMP tests (`subtype = "icmp"`) between 0 and 10.
 - `persist_cookies` (Boolean) Persist cookies across redirects.
+- `plain_proto_file` (String) The content of a proto file as a string
```

Comment:
Add period at the end of this line for consistency

Reply:
I made another pr https://github.com/DataDog/terraform-provider-datadog/pull/2274 for these comment, feel free to give it a 👍